service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/Microservice.java

```diff
@@ -70,6 +70,8 @@
   @JsonIgnore
   private MicroserviceInstance instance;
 
+  private boolean thirdPartyService;
```

Comment:
currently, SC not support this
need to comment

Reply:
Comment added, please review it